code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
The console is horrible, non NanoUI... and I suspect there is probably benefits to reducing the number of un-used objects... which with our server pop... it is undoubtedly the case.